modules/social_features/social_follow_user/social_follow_user.module

```diff
@@ -162,9 +172,23 @@ function social_follow_user_preprocess_profile(array &$variables): void {
         $variables['following'] = TRUE;
```

Comment:
@volodymyr-sydor I see we already have this `following` variable. Where it is used?
Do we really need extra `disable_following` variable?

Reply:
Yes, because followers/following variables uses to indicate which label we should use on the different followers/following list
but in current case we should have addition flag to check follow privacy rule

Reply:
Ok